src/AwsClient.php

```diff
@@ -407,45 +414,34 @@ private function addSignatureMiddleware()
     {
         $api = $this->getApi();
         $provider = $this->signatureProvider;
-        $version = $this->config['signature_version'];
+        $signatureVersion = $this->config['signature_version'];
         $name = $this->config['signing_name'];
         $region = $this->config['signing_region'];
 
         $resolver = static function (
             CommandInterface $c
-        ) use ($api, $provider, $name, $region, $version) {
+        ) use ($api, $provider, $name, $region, $signatureVersion) {
             if (!empty($c['@context']['signing_region'])) {
                 $region = $c['@context']['signing_region'];
             }
             if (!empty($c['@context']['signing_service'])) {
                 $name = $c['@context']['signing_service'];
             }
+            if (!empty($c['@context']['signature_version'])) {
+                $signatureVersion = $c['@context']['signature_version'];
+            }
 
             $authType = $api->getOperation($c->getName())['authtype'];
-            switch ($authType){
+            switch ($authType) {
                 case 'none':
-                    $version = 'anonymous';
+                    $signatureVersion = 'anonymous';
                     break;
                 case 'v4-unsigned-body':
-                    $version = 'v4-unsigned-body';
+                    $signatureVersion = 'v4-unsigned-body';
                     break;
-                case 'bearer':
```

Comment:
I may have missed it, but where did resolving a `bearer` entry land?

Reply:
As far as I'm aware, there hasn't been a service that has modeled `bearer` in an operation's `authtype`, but looking at the bearer token spec I see it's supported.  I figured multi-auth would render this useless, but I'll re-add it just in case